### ISSUE TITLE
Show type(manual/automated) for tests

### ISSUE THREAD

**Issue by mrajashree** [rancher/cis-operator#45]
- [x] Show type (manual/automated) and scored/not-scored in the scan result

~- [ ] Update UI to show the same~

**Comment by mrajashree** [comment]
We decided we don't need these columns in the UI, they have been added to the report
cc @deniseschannon

**Comment by mrajashree** [comment]
There will be two new columns in the report, `test_type` and `scored`

**Comment by sowmyav27** [comment]
**On 2.5-head - commit id: `f671292`**

- Repo: `https://github.com/mrajashree/charts` 
- branch: `cis16`
- Run scan using profile - `rke-profile-permissive`.
- Scan's detail page does NOT have columns `test_type` and `scored`.
- Download Scan report
- The .csv file has columns `test_type` and `scored`.
- test_type has values: `skip`, `manual` and `<blank>` (no value)
- Scored column has value `True` and `False`

**Comment by sowmyav27** [comment]
Reopening the issue to fix `test_type` to show `Automated`

**Comment by mrajashree** [comment]
It was decided offline to not add the type Automated. But one change needed is, all tests that have the word "Manual" in text field should have type:manual marked

**Comment by sowmyav27** [comment]
@mrajashree The following tests show "Scored" in the description. Is this expected? Should it be renamed Automated? 

```
1.1.11 | Ensure that the etcd data directory   permissions are set to 700 or more restrictive (Scored)
1.1.12 | Ensure that the etcd data directory ownership is set to etcd:etcd   (Scored)
1.1.19 | Ensure that the Kubernetes PKI directory and file ownership is set to   root:root (Scored)
1.1.20 | Ensure that the Kubernetes PKI certificate file permissions are set to   644 or more restrictive (Scored)
1.1.21 | Ensure that the Kubernetes PKI key file permissions are set to 600   (Scored)
```

This is an rke-cis 1.6 permissive scan run on a cluster on 2.5-head using latest charts.
[scan-dvv6j-dnbxr--2020-12-12-145140.csv.zip](https://github.com/rancher/cis-operator/files/5683909/scan-dvv6j-dnbxr--2020-12-12-145140.csv.zip)

**Comment by mrajashree** [comment]
Available to test with security-scan RC v0.2.2-rc3

**Comment by sowmyav27** [comment]
**On 2.5-head - commit id: `baf4867`**

- Using cis v2 app v- `1.0.3` and  security-scan RC `v0.2.2-rc3`
- Deploy CIS v2
- Run a CIS 1.6 permissive scan
- https://github.com/rancher/cis-operator/issues/45#issuecomment-743918660 issue is seen fixed. The tests are now marked `Automated`
- All tests having "Manual" have test_type: manual
- All tests having "Manual" , with status: "Not Applicable" have test_type: skip
